v1/operator.go

```diff
@@ -0,0 +1,135 @@
+package v1
+
+import (
+	"context"
+
+	client "github.com/hashicorp/nomad-openapi/clients/go/v1"
+)
+
+type Operator struct {
+	client *Client
+}
+
+func (c *Client) Operator() *Operator {
+	return &Operator{client: c}
+}
+
+func (o *Operator) OperatorApi() *client.OperatorApiService {
+	return o.client.apiClient.OperatorApi
+}
+
+func (o *Operator) Raft(ctx context.Context) (*client.RaftConfiguration, error) {
+	request := o.OperatorApi().GetOperatorRaftConfiguration(o.client.Ctx)
+
+	result, err := o.client.ExecRequest(ctx, request)
+	if err != nil {
+		return nil, err
+	}
+
+	final := result.(client.RaftConfiguration)
+	return &final, nil
+}
+
+func (o *Operator) Peer(ctx context.Context) error {
+	request := o.OperatorApi().DeleteOperatorRaftPeer(o.client.Ctx)
+
+	err := o.client.ExecNoResponseRequest(ctx, request)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (o *Operator) Autopilot(ctx context.Context) (*client.AutopilotConfiguration, error) {
+	request := o.OperatorApi().GetOperatorAutopilotConfiguration(o.client.Ctx)
+
+	result, err := o.client.ExecRequest(ctx, request)
+	if err != nil {
+		return nil, err
+	}
+
+	final := result.(client.AutopilotConfiguration)
+	return &final, nil
+}
+
+func (o *Operator) UpdateAutopilot(
```

Comment:
I feel like this probably needs to accept an AutopilotConfiguration struct instead of a large set of fields, similar to how `Job.PlanOpts` works. Is that possible?

Reply:
done!